great_expectations_cloud/agent/actions/generate_schema_change_expectations_action.py

```diff
@@ -36,6 +38,32 @@
 LOGGER.setLevel(logging.DEBUG)
 
 
+class PartialSchemaChangeExpectationError(GXAgentError):
+    def __init__(self, asset_to_error_map: dict[str, Exception], assets_attempted: int):
+        self.MAX_ERRORS_TO_DISPLAY = 5
+        self.asset_to_error_map = asset_to_error_map
+        num_assets_with_errors = len(self.asset_to_error_map)
+        self.message = f"Failed to generate schema change expectations for {num_assets_with_errors} of the {assets_attempted} assets."
+        super().__init__(self._build_error_message())
+
```

Comment:
going from `LOGGER.warning(f"asset_name: {asset_name} failed with error: {e}")` to this in a day is so cool to me. Thanks again @anthonyburdi :)